nextflow_schema.json

```diff
@@ -516,6 +516,29 @@
                     "fa_icon": "fas fa-file",
                     "description": "The desired output format.",
                     "enum": ["tsv", "csv", "arrow", "parquet", "biom"]
+                },
+                "taxpasta_taxonomy_dir": {
+                    "type": "boolean",
```

Comment:
```suggestion
                    "type": "string",
```

Reply:
I'm not sure if any other json object entries are required here for string (possibly a default? Check the other path-like parameters)

Reply:
The other path-like parameters are strings.